tests/beaker_tests/cisco_bgp/test_bgp_non_defaults.rb

```diff
@@ -116,6 +113,13 @@
   'timer_bgp_holdtime'                     => '110',
 }
 
+expected_values_diff = {
```

Comment:
Similar approach - make the platform a parameter to the method, and define part of the string depending on the platform:

https://github.com/cisco/cisco-network-puppet-module/blob/feature/grpc/tests/beaker_tests/bgp/bgplib.rb#L37
https://github.com/cisco/cisco-network-puppet-module/blob/feature/grpc/tests/beaker_tests/bgp/bgplib.rb#L40
https://github.com/cisco/cisco-network-puppet-module/blob/feature/grpc/tests/beaker_tests/bgp/bgplib.rb#L86